.github/CODEOWNERS

```
@@ -61,9 +61,6 @@ Python/traceback.c            @iritkatriel
 /Tools/build/parse_html5_entities.py   @ezio-melotti
 
 # Import (including importlib).
-# Ignoring importlib.h so as to not get flagged on
-# all pull requests that change the emitted
-# bytecode.
 **/*import*.c                 @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
```

re: https://github.com/python/cpython/commit/811b2878dfc68dc3ecd81fb4b370c6e1f9ec69c2 / https://github.com/python/cpython/pull/5087:

Can we merge these two back together? (GitHub isn't let me do a multiline suggestion, so this is part 1)

```suggestion
**/*import*                   @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
```